requirements.txt

```diff
@@ -0,0 +1,4 @@
+myst-parser
```

Comment:
The myst-parser extension is used to support markdown source files. Currently, the only markdown file is `docs/contributing.md`. We can remove the myst-parser dependency if we convert contributing.md from markdown to rst.